### ISSUE TITLE
Performance on the KITTI Dataset

### ISSUE THREAD

**Issue by Magicboomliu** [WU-CVGL/OmniScene#30]
Thank you for your work. I noticed from your public codebase that you have experimented with the KITTI dataset. Unlike nuScenes, KITTI provides only two perspective cameras by default. When I trained with a bin size of 8 using stereo images as input, the resulting FeedForwardGS reconstructions were quite blurry and ambiguous.   

![Image](https://github.com/user-attachments/assets/5f6cf650-6689-4774-8338-8e3ab82b79a1)

Have you encountered similar issues in your experiments? If so, do you have any suggestions for improvement—such as adjustments to hyperparameters or model settings?

**Comment by wswdx** [comment]
> Thank you for your work. I noticed from your public codebase that you have experimented with the KITTI dataset. Unlike nuScenes, KITTI provides only two perspective cameras by default. When I trained with a bin size of 8 using stereo images as input, the resulting FeedForwardGS reconstructions were quite blurry and ambiguous.
> 
> ![Image](https://github.com/user-attachments/assets/5f6cf650-6689-4774-8338-8e3ab82b79a1)
> 
> Have you encountered similar issues in your experiments? If so, do you have any suggestions for improvement—such as adjustments to hyperparameters or model settings?

Sorry. I have never tried experiments on KITTI before. Where did get the information?
I am not sure about your settings on KITTI dataset. As I know, KITTI has two forwarding cameras located near to each other. So, the overlap between images captured by the two cameras is large. I think it is enough to use only one image rather than stereo images in your experiments.

**Comment by Magicboomliu** [comment]
Thanks for your feedback!  I am  now working on feedforwardGS on KITTI, if you are interested, I am glad to learn from you and have a coorp.

**Comment by Magicboomliu** [comment]
Sorry for one more question, in the `pixel_gs.py`  : 
https://github.com/WU-CVGL/Omni-Scene/blob/main/model/pixel/pixel_gs.py#L153 

```
        # rearrange pseudo depths and confs
        depths_in = rearrange(depths_in, "b v h w -> (b v) () h w")
        confs_in = rearrange(confs_in, "b v h w -> (b v) () h w")
        img_feats = torch.cat([img_feats, depths_in / 20.0, confs_in], dim=1)

```
it seems that you use the depth result from the Metric3D-v2 for inference,(concated with the RGB), can you tell me why here the depth is divided by 20?

**Comment by wswdx** [comment]
> Sorry for one more question, in the `pixel_gs.py` : https://github.com/WU-CVGL/Omni-Scene/blob/main/model/pixel/pixel_gs.py#L153
> 
> ```
>         # rearrange pseudo depths and confs
>         depths_in = rearrange(depths_in, "b v h w -> (b v) () h w")
>         confs_in = rearrange(confs_in, "b v h w -> (b v) () h w")
>         img_feats = torch.cat([img_feats, depths_in / 20.0, confs_in], dim=1)
> ```
> 
> it seems that you use the depth result from the Metric3D-v2 for inference,(concated with the RGB), can you tell me why here the depth is divided by 20?

You can just ignore it. I just did this for kinda numerical normalization.

**Comment by Magicboomliu** [comment]
Thanks for your in-time feedback. Nice Work!
